pyvista/core/dataset.py

```diff
@@ -2523,3 +2524,49 @@ def cell_type(self, ind: int) -> int:
 
         """
         return self.GetCellType(ind)
+
+    def cell_is_point_inside(self, ind: int, point: Vector) -> bool:
```

Comment:
```suggestion
    def point_is_inside_cell(self, ind: int, point: Vector) -> bool:
```
I guess this is more clear what the method is doing.

Reply:
Thanks, this is indeed better.  I was trying to be thematically similar to the `DataSet.cell_*` methods like `DataSet.cell_bounds` as it indicates it is operating on a single cell basis.  But I think you name is the most clear.  We can have a `See Also` section potentially to address my thinking.

I will clean up the usage after committing your suggestion directly.